FitNesseRoot/DbFit/AcceptanceTests/JavaTests/InformixTests/FlowMode/InformixMode/SetUp/content.txt

```diff
@@ -0,0 +1,3 @@
+!|dbfit.InformixTest|
+
+|Connect|localhost:16466|dbfit|DBFIT|dbfit_ifx|
```

Comment:
Good question... If we want to test `connect` variants we may use some dummy/very simple just for that purpose.